docs/dev/check_references.md

```diff
@@ -1,11 +1,28 @@
 ---
 title: Integration Assets Reference
 kind: documentation
+description: Learn about the various assets that need to be included when preparing a Datadog integration.
+further_reading:
+  - link: "https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md"
+    tag: "GitHub"
+    text: "Contributing Guidelines for the Documentation Site"
+  - link: "/developers/integrations/"
+    tag: "Documentation"
+    text: "Learn about creating an Agent or API-based integration"
+  - link: "/developers/integrations/oauth_for_integrations/"
+    tag: "Documentation"
+    text: "Learn about using OAuth for integrations"
 ---
 
+## Overview
+
+This page walks you through how to develop an offering on the [Integrations page][12]. 
```

Comment:
This also applies to Marketplace, could we add that in here as well (with a link)?

```suggestion
This page walks you through how to develop an offering on the [Integrations page][12] or Datadog Marketplace. 
```